pkg/controller/jobs/jobset/jobset_controller.go

```diff
@@ -223,7 +223,7 @@ func (j *JobSet) SetManagedBy(managedBy *string) {
 	j.Spec.ManagedBy = managedBy
 }
 
-func podsCountPerReplica(rj *jobsetapi.ReplicatedJob) int32 {
+func PodsCountPerReplica(rj *jobsetapi.ReplicatedJob) int32 {
```

Comment:
Hope we can revert that after https://github.com/kubernetes-sigs/kueue/pull/6997